build.gradle

```
@@ -84,3 +89,15 @@ jar {
     }
     dependsOn(copyToLib)
 }
+
+nexusPublishing {
+    packageGroup = 'org.rundeck-plugins'
+    repositories {
+        sonatype{
+            stagingProfileId = '67d196ce5bae'
+        }
+    }
+}
+
```

Tests will not run without this 
```suggestion

test {
    useJUnitPlatform()
}
```